### PR TITLE
Simplify pdf2txt

### PR DESCRIPTION
**Pull request by hason**
Remove duplicate pieces of code.

### CODE REVIEW

File: tools/pdf2txt.py [hunk 1]
```diff
@@ -39,15 +44,8 @@ def extract_text(files=[], outfile='-',
     else:
         laparams = None
 
-    imagewriter = None
-    if output_dir:
-        imagewriter = ImageWriter(output_dir)
-
```
Comment (hason) [inline tools/pdf2txt.py]: Duplicate https://github.com/pdfminer/pdfminer.six/blob/develop/pdfminer/high_level.py#L65-L67

File: tools/pdf2txt.py [hunk 2]
```diff
@@ -145,33 +142,19 @@ def main(args=None):
     if A.pagenos:
         A.page_numbers = set([int(x)-1 for x in A.pagenos.split(",")])
 
-    imagewriter = None
-    if A.output_dir:
-        imagewriter = ImageWriter(A.output_dir)
-
```
Comment (hason) [inline tools/pdf2txt.py]: Duplicate https://github.com/pdfminer/pdfminer.six/blob/develop/pdfminer/high_level.py#L65-L67

File: tools/pdf2txt.py [hunk 3]
```diff
@@ -145,33 +142,19 @@ def main(args=None):
...
                 A.output_type = alttype
 
-    if A.outfile == "-":
-        outfp = sys.stdout
-        if outfp.encoding is not None:
-            # Why ignore outfp.encoding? :-/ stupid cathal?
-            A.codec = 'utf-8'
-    else:
-        outfp = open(A.outfile, "wb")
-
```
Comment (hason) [inline tools/pdf2txt.py]: Duplicate https://github.com/pdfminer/pdfminer.six/blob/develop/tools/pdf2txt.py#L54-L59

File: tools/pdf2txt.py (hunk 1 above)
Comment (pietermarsman) [reply tools/pdf2txt.py]: It took me a while to understand that it is not only duplicate code (hence, a function could be extracted), but also that the `imagewriter` variable is unused.

**Comment by pietermarsman** [comment]
@hason Thanks for noticing and fixing!

**Pull request merged**